axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+
+In case of a `SubscribingEventProcessor`, the invocation happens during command execution.
+No new root trace will be created, but the inner process span of the processor is added to indicate the invocation.
+
+### Deadlines
+
+Any action related to deadlines is traced in order to gain insight into what happened during specific calls. 
+Mutations on deadlines generally happen from another root trace, such as a command or saga.
+The handling span of a deadline will be linked to the scheduling span for easy navigation.
+
+| Trace name                                                               | Description                                                                |
+|--------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| `${DeadlineManagerClass}.schedule(${deadlineName})`                      | A deadline was scheduled.                                                  |
+| `${DeadlineManagerClass}.cancelSchedule(${deadlineName}, ${scheduleId})` | A deadline was cancelled based on name and scheduleId.                     |
+| `${DeadlineManagerClass}.cancelAll(${deadlineName})`                     | All deadlines with a specific name were cancelled.                         |
+| `${DeadlineManagerClass}.cancelAllWithinScope(${deadlineName})`          | All deadlines within a specific scope with a specific name were cancelled. |
+| `DeadlineJob.execute(${deadlineName},${DeadlinePayloadClass})`           | Root trace of a deadline firing, containing the name and payload class.    |
+
+### Snapshotting
+
+Snapshotting is done in a separate root trace, due to the fact that it's an asynchronous action and has no user impact.
+However, it can still be useful to measure the performance of snapshotting and see when it is triggered.
+The root trace of the `Snapshotter` invocation will be linked to the command handling span 
+after which the snapshot was scheduled to be created.
+
+| Trace name                                                                  | Description                                                                                                           |
+|-----------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| `${SnapshotterClass}.createSnapshot($aggregateClass)`                       | A snapshot creation task is being submitted. Depending on performance, the executor might take a while to pick it up. |
+| `${SnapshotterClass}.createSnapshot($aggregateClass, $aggregateIdentifier)` | The `Snapshotter` is now creating the snapshot.                                                                       |
+
+Note that the root trace does not contain the aggregate identifier,
+so any snapshot invocation of the same aggregate type is grouped together by the APM tool.
```

Comment:
Nit: I'd make a separate sentence out of "so any snapshot..." for clarity..

Reply:
I changed the sentence, I kind of like it in the new form. It feels wrong to split up